scripts/convert_ipynb_to_mdx.py

```diff
@@ -0,0 +1,1080 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import io
+import json
+import os
+import re
+import shutil
+import subprocess
+import uuid
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import mdformat  # @manual=fbsource//third-party/pypi/mdformat:mdformat
```

Comment:
This seems misplaced in OSS code. You can probably add it directly to the targets file
```suggestion
import mdformat
```